multiqc_sarek_project.sh

```diff
@@ -17,7 +17,7 @@ PROJECT_PATH=$1
 PROJECT_ID=$(basename $PROJECT_PATH)
 REPORT_FILENAME=$PROJECT_ID"_multiqc_report"
 REPORT_FILENAME_QC=$PROJECT_ID"_multiqc_report_qc"
-SCRIPTS_DIR="/proj/ngi2016001/private/scripts"
+SCRIPTS_DIR="$(readlink -f "$(dirname $0)")"
 CONFIG_DIR=$SCRIPTS_DIR"/config"
 
```

Comment:
what about adding:

`module load bioinfo-tools MultiQC`

do you think it would interfere with the desired environment? I know I sometimes have problems with not having MultiQC in the path when the script is submitted..